### PR TITLE
1594 cloudflare imagees

### PR DESCRIPTION
**Pull request by roiLeo**
### PR type

- [x] Bugfix
- [x] Feature

### What's new?

- [x] model viewer in gallery

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Had issue bounty label ?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DVYy1qnocE8t6ZvUfPx3rEjG829khNRXx3YrCGVHHj19Lcb)

### Screenshot
Capture d’écran 2022-01-31 à 12 30 55 PM

### CODE REVIEW

File: components/rmrk/Gallery/Gallery.vue [hunk 1]
```diff
@@ -29,9 +29,16 @@
                   }}</span>
                 </span>
                 <BasicImage
+                  v-show="nft.image"
                   :src="nft.image"
                   :alt="nft.name"
                   customClass="gallery__image-wrapper" />
+
+                <Yolo
```
Comment (roiLeo) [inline components/rmrk/Gallery/Gallery.vue]: wut?

File: components/rmrk/Gallery/Gallery.vue [hunk 2]
```diff
@@ -140,14 +151,18 @@ export default class Gallery extends mixins(PrefixMixin) {
     )
   }
 
+  get enableAllArtworks(): boolean {
+    return this.$store.getters['preferences/getLoadAllArtwork']
```
Comment (roiLeo) [inline components/rmrk/Gallery/Gallery.vue:155]: naming `getLoadAllArtwork`?

File: components/rmrk/Media/ModelMedia.vue [hunk 3]
```diff
@@ -1,16 +1,15 @@
...
-      id="reveal"
       class="view-model__component"
       :src="src"
       :poster="poster"
       :alt="description"
       :availableAnimations="availableAnimations"
       auto-rotate
-      camera-controls
-      ar
-      ar-modes="webxr scene-viewer quick-look"
```
Comment (roiLeo) [inline components/rmrk/Media/ModelMedia.vue:13]: include it back?

File: components/rmrk/Gallery/Gallery.vue (hunk 1 above)
Comment (vikiival) [reply components/rmrk/Gallery/Gallery.vue]: Let me push the changes

**Comment by roiLeo** [comment]
> @roiLeo may post your KSM address so we can process a bounty

updated description

**Comment by yangwao** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 157.49 USD/KSM ~ 0.635 $KSM
🧗 [DVYy1qnocE8t6ZvUfPx3rEjG829khNRXx3YrCGVHHj19Lcb](https://kusama.subscan.io/account/DVYy1qnocE8t6ZvUfPx3rEjG829khNRXx3YrCGVHHj19Lcb)
🔗 [0x5a1bb8e427a61e5e90212d6dafd081cd0e2fc06d75800a76c982906cae4d8904](https://kusama.subscan.io/extrinsic/0x5a1bb8e427a61e5e90212d6dafd081cd0e2fc06d75800a76c982906cae4d8904)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**